Source/Schema.NET/DateTimeHelper.cs

```diff
@@ -18,8 +18,13 @@ internal static class DateTimeHelper
         /// </summary>
         /// <param name="input">The input string</param>
         /// <returns>True if the input string has an offset defined</returns>
-        public static bool ContainsTimeOffset(string input)
+        public static bool ContainsTimeOffset(string? input)
         {
+            if (input is null)
```

Comment:
I made the methods in this helper handle nulls but I wonder if I should have handled null further up and then these can go back to be non-nullable.

Reply:
This looks like it follows the behaviour of other `Try*` methods that we call in our `ValuesJsonConverter` so I think you made the right decision.